src/views/CreatePage.vue

```diff
@@ -288,6 +299,10 @@ const generateAndSaveThumbnail = () => {
     },
   )
 }
+
+const resetCanvasFor3PointLeader = async () => {
+  await useStoreCanvas().resetCanvas(stageParentDiv.value)
+}
```

Comment:
```suggestion
const resetCanvasFor3PointLeader = () => {
  useStoreCanvas().resetCanvas()
}
```
@clickで直接呼び出してもいいと思います。